core/templates/dev/head/components/forms/schema-based-editors/schema-based-html-editor.directive.html

```diff
@@ -1,8 +1,8 @@
 <!-- label-for-focus-target feature is removed since CKEditor has a
 focusManager to manage the focus activity automatically. -->
-<ck-editor-rte ng-if="!$ctrl.isDisabled()" ng-model="$ctrl.localValue"
-               ui-config="$ctrl.uiConfig()">
-</ck-editor-rte>
+<ck-editor-4-rte ng-if="!$ctrl.isDisabled()" ng-model="$ctrl.localValue"
+                 ui-config="$ctrl.uiConfig()">
```

Comment:
This can go on the above line right?

Reply:
The line length exceeds, also this is a stacked PR so we'll be adding few more attributes to this Thanks!